releases/release-7.0.0.md

```diff
@@ -0,0 +1,468 @@
+
+    For more information, see [documentation](/partitioned-table.md#key-partitioning).
+
+### DB operations
+
+* TiCDC 支持 storage sink，可输出变更数据至 cloud storage (GA) [#6797](https://github.com/pingcap/tiflow/issues/6797) @[zhaoxinyu](https://github.com/zhaoxinyu) **tw:hfxsd**
+
+    TiCDC 支持将 changed log 输出到 Amazon S3、Azure Blob Storage、NFS，以及兼容 Amazon S3 协议的存储服务中。Cloud storage 价格便宜，使用方便。对于不使用 Kafka 的用户，可以选择使用 storage sink。使用该功能，TiCDC 会将 changed log 保存到文件，发送到存储系统中。用户自研的消费程序定时从存储系统读取新产生的 changed log 进行数据处理。
+
+    Storage sink 支持格式为 canal-json 和 csv 的 changed log。
+
+    For more information, see [documentation](https://docs.pingcap.com/zh/tidb/stable/ticdc-sink-to-cloud-storage).
+
+* TiCDC Open API V2 GA @[sdojjy](https://github.com/sdojjy) **tw:hfxsd**
+
+    TiCDC 提供 OpenAPI 功能，用户可以通过 OpenAPI v2 对 TiCDC 集群进行查询和运维操作。OpenAPI 的功能是 [`cdc cli` 工具](/ticdc/ticdc-manage-changefeed.md)的一个子集。用户可以通过 OpenAPI 完成 TiCDC 集群的如下运维操作：
+
+    - [获取 TiCDC 节点状态信息](#获取-ticdc-节点状态信息)
+    - [检查 TiCDC 集群的健康状态](#检查-ticdc-集群的健康状态)
+    - [创建同步任务](#创建同步任务)
+    - [删除同步任务](#删除同步任务)
+    - [更新同步任务配置](#更新同步任务配置)
+    - [查询同步任务列表](#查询同步任务列表)
+    - [查询特定同步任务](#查询特定同步任务)
+    - [暂停同步任务](#暂停同步任务)
+    - [恢复同步任务](#恢复同步任务)
+    - [查询同步子任务列表](#查询同步子任务列表)
+    - [查询特定同步子任务](#查询特定同步子任务)
+    - [查询 TiCDC 服务进程列表](#查询-ticdc-服务进程列表)
+    - [驱逐 owner 节点](#驱逐-owner-节点)
+    - [动态调整 TiCDC Server 日志级别](#动态调整-ticdc-server-日志级别)
+
+  For more information, see [documentation](https://github.com/pingcap/docs-cn/pull/13224).
+
+### Observability
+
+* 功能标题 [#issue号](链接) @[贡献者 GitHub ID](链接)
+
+    功能描述（需要包含这个功能是什么、在什么场景下对用户有什么价值、怎么用）
+
+    For more information, see [documentation](链接).
+
+### Security
+
+* 功能标题 [#issue号](链接) @[贡献者 GitHub ID](链接)
+
+    功能描述（需要包含这个功能是什么、在什么场景下对用户有什么价值、怎么用）
+
+    For more information, see [documentation](链接).
+
+### Data migration
+
+* Load data 语句集成 Lightning ，用户可以使用 Load data 命令完成原先需要单独使用 Lightning 才能完成的数据导入任务。    [#40499](https://github.com/pingcap/tidb/issues/40499) @[lance6716](https://github.com/lance6716) **tw:hfxsd**
+
+    在集成 Lightning 之前，Load data 语句只能用于导入位于客户端的数据文件，如果用户要从云存储导入数据，就得借助 Lightning 来实现。但是单独部署 Lightning 又会带来额外的部署成本和管理成本。将 Lightning 逻辑导入能力（TiDB backend ）集成到 Load data 命令后，不仅可以省去 Lightning 的部署和管理成本。还可以借助 Lightning 的功能大大扩展 load data 语句的能力。 部分增强的功能举例说明如下：
+
+    - 支持从 S3 导入数据到 TiDB，且支持通配符一次性匹配多个源文件导入到 TiDB 。
+    - 支持 CSV、TSV、Parquet、SQL(mydumper/dumpling) 格式的源文件。
+    - 支持 precheck ，可在导入之前将所有不满足导入数据的问题检测出来，用户根据检测结果优化后，再次提交任务。提升任务配置体验。
+    - 支持将任务设置为 Detached，让任务在后台执行。
+    - 支持任务管理，可通过 show load data jobid 查询任务状态和进展详情。方便用户管理和维护。
+
+  For more information, see [documentation](待补充).
```

Comment:
```suggestion
* The `LOAD DATA` statement integrates with TiDB Lightning, so you can use the `LOAD DATA` command to complete data import tasks that would otherwise be done using TiDB Lightning alone. [#40499](https://github.com/pingcap/tidb/issues/40499) @[lance6716](https://github.com/lance6716) **tw:hfxsd**

    Before integrating TiDB Lightning, the `LOAD DATA` statement could only be used to import data files located on the client side. If you wanted to import data from cloud storage, you had to rely on TiDB Lightning to do so. However, deploying TiDB Lightning separately would incur additional deployment and management costs. By integrating the TiDB Lightning logical import capability (TiDB Backend) into the `LOAD DATA` command, you can not only eliminate the deployment and management costs of TiDB Lightning, but also greatly extend the capabilities of the `LOAD DATA` statement with the TiDB Lightning features. Some examples of the enhanced functionality are as follows:

    - Supports importing data from Amazon S3 and Google Cloud Storage to TiDB. Supports importing multiple source files to TiDB in one go with wildcards.
    - Support `DEFINED NULL BY` to define null.
    - Support for source files in CSV, TSV, Parquet, and SQL (mydumper/dumpling) formats.
    - Support setting tasks as `Detached` to let tasks run in the background.
    - Support task management. You can query task status and progress details by `SHOW LOAD DATA jobid`, which is convenient for management and maintenance.
    
    For more information, see [documentation](/sql-statements/sql-statement-load-data.md).
```